constellations/orion/src/controllers/publish_controller.go

```diff
@@ -19,38 +19,18 @@ func GetAllUnpublished(c *gin.Context) {
 }
 
 func callGetUnpublishedRepos() (domains.UnpublishedDomains, error) {
-	programList, err := repos.ProgramRepo.SelectAllUnpublished()
-	if err != nil {
-		return domains.UnpublishedDomains{}, err
-	}
 	classList, err := repos.ClassRepo.SelectAllUnpublished()
 	if err != nil {
 		return domains.UnpublishedDomains{}, err
 	}
-	locationList, err := repos.LocationRepo.SelectAllUnpublished()
-	if err != nil {
-		return domains.UnpublishedDomains{}, err
-	}
 	achieveList, err := repos.AchieveRepo.SelectAllUnpublished()
 	if err != nil {
 		return domains.UnpublishedDomains{}, err
 	}
-	semesterList, err := repos.SemesterRepo.SelectAllUnpublished()
-	if err != nil {
-		return domains.UnpublishedDomains{}, err
-	}
-	sessionList, err := repos.SessionRepo.SelectAllUnpublished()
-	if err != nil {
-		return domains.UnpublishedDomains{}, err
-	}
 
 	unpublishedDomains := domains.UnpublishedDomains{
-		Programs:  programList,
-		Classes:   classList,
-		Locations: locationList,
-		Achieves:  achieveList,
-		Semesters: semesterList,
-		Sessions:  sessionList,
+		Classes:  classList,
+		Achieves: achieveList,
```

Comment:
should achieveList also get removed? Looks like achievements also have a PublishedAt column